hw06-atm/src/main/java/ru/otus/atm/cell/StandardCellRepository.java

```diff
@@ -0,0 +1,53 @@
+package ru.otus.atm.cell;
+
+import ru.otus.atm.banknote.Banknote;
+import ru.otus.atm.exceptions.IllegalBanknoteException;
+import ru.otus.atm.exceptions.NegativeAmountException;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class StandardCellRepository implements CellRepository {
+    private final Map<Banknote, Integer> cells = new TreeMap<>(Comparator.comparingInt(Banknote::getValue).reversed());
```

Comment:
Раз уж у вас `CellRepository`, то можно ввести понятие ячейки и хранить коллекцию `Cell`.
```suggestion
    private final List<Cell> cells = ...;
```

Подумайте самостоятельно, что будет внутри `Cell`

Reply:
Сделал

Reply:
Сделал